DOCKER/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM golang:1.17-alpine as builder
 RUN apk update && \
     apk upgrade && \
-    apk --no-cache add make
+    apk --no-cache add make git
```

Comment:
Why is git needed here?

Reply:
Git is needed here because when we build binary with make it uses git to get the actual version. In the beginning of the [Makefile](https://github.com/tendermint/tendermint/blob/41a1bf539bd9e3f312e5a52d2424de41d646be21/Makefile#L7-L14) we have the [LD_FLAGS](https://github.com/tendermint/tendermint/blob/41a1bf539bd9e3f312e5a52d2424de41d646be21/Makefile#L14) variable setting which allows us to change the variable in the source code to set the actual tendermint version based on the git information. If git was not found during make running, the value of $(VERSION) will be empty, and because of this `tendermint version` will return an empty string instead of correct value. This is the reason why we need to install git in docker before building the tendermint binary.